### PR TITLE
feat: user query

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

File: internal/admin/repository/eventsourcing/handler/handler.go [hunk 1]
```diff
@@ -32,11 +32,7 @@ func (h *handler) Eventstore() v1.Eventstore {
 }
 
 func Register(configs Configs, bulkLimit, errorCount uint64, view *view.View, es v1.Eventstore, defaults systemdefaults.SystemDefaults, command *command.Commands, static static.Storage, localDevMode bool) []query.Handler {
-	handlers := []query.Handler{
-		newUser(
-			handler{view, bulkLimit, configs.cycleDuration("User"), errorCount, es},
-			defaults),
-	}
+	handlers := []query.Handler{}
```
Comment (fgerschwiler) [inline internal/admin/repository/eventsourcing/handler/handler.go:35]: do we still need this if we have no handlers to register?

File: internal/api/grpc/auth/profile.go [hunk 2]
```diff
@@ -3,18 +3,19 @@ package auth
...
 
 func (s *Server) GetMyProfile(ctx context.Context, req *auth_pb.GetMyProfileRequest) (*auth_pb.GetMyProfileResponse, error) {
-	profile, err := s.repo.MyProfile(ctx)
+	profile, err := s.query.GetHumanProfile(ctx, authz.GetCtxData(ctx).UserID)
 	if err != nil {
 		return nil, err
 	}
 	return &auth_pb.GetMyProfileResponse{
-		Profile: user_grpc.ProfileToPb(profile),
+		Profile: user_grpc.ProfileToPb(profile, "prefix"), //TODO: prefix,
```
Comment (fgerschwiler) [inline internal/api/grpc/auth/profile.go:18]: what is this prefix?

File: internal/api/grpc/auth/user.go [hunk 3]
```diff
@@ -17,11 +17,11 @@ import (
 )
 
 func (s *Server) GetMyUser(ctx context.Context, _ *auth_pb.GetMyUserRequest) (*auth_pb.GetMyUserResponse, error) {
-	user, err := s.repo.MyUser(ctx)
+	user, err := s.query.GetUserByID(ctx, authz.GetCtxData(ctx).UserID)
 	if err != nil {
 		return nil, err
 	}
-	return &auth_pb.GetMyUserResponse{User: user_grpc.UserToPb(user)}, nil
+	return &auth_pb.GetMyUserResponse{User: user_grpc.UserToPb(user, "prefix")}, nil //TODO: prefix
```
Comment (fgerschwiler) [inline internal/api/grpc/auth/user.go:24]: what is this prefix?

File: internal/query/user.go [hunk 4]
```diff
@@ -122,7 +218,693 @@ var (
...
+			HumanPhoneCol.identifier(),
+			HumanIsPhoneVerifiedCol.identifier()).
+			From(userTable.identifier()).
+			LeftJoin(join(HumanUserIDCol, UserIDCol)).
+			PlaceholderFormat(sq.Dollar),
+		func(row *sql.Row) (*Phone, error) {
+			e := new(Phone)
+
+			humanID := sql.NullString{}
+			email := sql.NullString{}
```
Comment (fgerschwiler) [inline internal/query/user.go:692]:
```suggestion
			phone := sql.NullString{}
```

File: internal/admin/repository/eventsourcing/handler/handler.go (hunk 1 above)
Comment (livio-a) [reply internal/admin/repository/eventsourcing/handler/handler.go:35]: there's still the styling handler

**Review by fgerschwiler — APPROVED** [review]

**Pull request merged**